package.json

```diff
@@ -23,14 +23,16 @@
     "lit-element": "^2.5.1"
   },
   "peerDependencies": {
-    "@alaskaairux/design-tokens": "^3.0.0",
-    "@alaskaairux/webcorestylesheets": "^3.0.0",
+    "@alaskaairux/design-tokens": "^3.3.0",
```

Comment:
```suggestion
```

Reply:
I will updated. thank

Reply:
For your information, I'm also keeping this css with the old `@alaskaairux/design-tokens` in the other component that have this because I'm afraid that would broke the UI.